src/constants/index.ts

```diff
@@ -38,6 +38,10 @@ export const colors = {
             sun: '#959EAB',
             moon: '#fff',
         },
+        orderBook: {
+            asks: 'var(--asks-level-4)',
```

Comment:
please, change all hardcoded values in this file to variables as you did here :)

Reply:
in next pr when i ll be move svg to tsx